### ISSUE TITLE
database error when a supervisor clicks on a case manager

### ISSUE THREAD

**Issue by tmaybe** [slco-2016/clientcomm#373]
When a supervisor-class user clicks on a case manager, they see a database error:

```
error: select * from "cms" where "department_id" = $1 - column "department_id" does not exist
    at Connection.parseE (/home/ubuntu/clientcomm/node_modules/pg/lib/connection.js:539:11)
    at Connection.parseMessage (/home/ubuntu/clientcomm/node_modules/pg/lib/connection.js:366:17)
    at Socket.<anonymous> (/home/ubuntu/clientcomm/node_modules/pg/lib/connection.js:105:22)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Socket.wrapped (/home/ubuntu/clientcomm/node_modules/newrelic/lib/transaction/tracer/index.js:155:28)
    at Socket.wrappedEmit [as emit] (/home/ubuntu/clientcomm/node_modules/newrelic/lib/transaction/tracer/index.js:187:46)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:543:20)
    at TCP.wrapped [as onread] (/home/ubuntu/clientcomm/node_modules/newrelic/lib/transaction/tracer/index.js:155:28)
```

**Comment by tmaybe** [comment]
also seen at `/org/clients`

**Comment by tmaybe** [comment]
fixed and tested with fd705629bf1c75e0c538a253e4adc17a042a62c0
